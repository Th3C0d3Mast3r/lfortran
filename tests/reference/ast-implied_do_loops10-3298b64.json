{
    "basename": "ast-implied_do_loops10-3298b64",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/implied_do_loops10.f90",
    "infile_hash": "2efdc481b725ae1203d010683681aa7cefaa4284865b8a440a16e133",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "ast-implied_do_loops10-3298b64.stderr",
    "stderr_hash": "bb9aacdaf1bd4cbb018e279be39b7f73f0c7c48f6a8ef32722d90fae",
    "returncode": 2
}