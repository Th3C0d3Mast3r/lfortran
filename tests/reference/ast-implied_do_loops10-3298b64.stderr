syntax error: Token ',' is unexpected here
 --> tests/../integration_tests/implied_do_loops10.f90:8:37
  |
8 |     data (colours%rgb(i), i=1, 3) /1, 2, 3/
  |                                     ^ 
